### PR TITLE
Creates basic app with core activities

### PR DESCRIPTION
**Pull request by rachelbock**
- Creates the shell for the application.
- Creates the home page activity with 4 links to the 4 activities.
- Creates the Action Bar Fragment which hosts the 4 buttons to access the 4 activities.
- Creates a placeholder for the Projects Activity.
- Creates a placeholder for the Wall Activity.
- Creates a placeholder for the UserInfoActivity.
- Adds images to drawables folder that could be used in the application.
- Creates a DialogFragment to collect basic User Information in User Activity.
- Creates layout files for each of the activities.
- Added description to fragment action on HomePage.
- Added description to fragment action on ProjectsActivity.
- Added description and TODO task to UserActivity.
- Added description to fragment action on WallActivity.
- Added method descriptions to ActionBarFragment.
- Added method descriptions and TODO tasks to DialogFragment.

### CODE REVIEW

File: app/src/main/java/com/rage/clamber/Activities/HomePage.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.rage.clamber.Activities;
+
+import android.os.Bundle;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+
+import com.rage.clamber.Fragments.ActionBarFragment;
+import com.rage.clamber.R;
+
+public class HomePage extends AppCompatActivity {
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Activities/HomePage.java:14]: It's probably a good idea to add a javadoc comment on the class itself as well, describing overall what it's for and what it's expected to do at a high level.

File: app/src/main/java/com/rage/clamber/Fragments/ActionBarFragment.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    }
+
+    /**
+     * Standard method to create a new ActionBar Fragment. This method is called
+     * within the various activities to set the Fragment to the top of their layout.
+     * @return returns the ActionBarFragment.
+     */
+    public static ActionBarFragment newInstance () {
+        ActionBarFragment fragment = new ActionBarFragment();
+        Bundle args = new Bundle();
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Fragments/ActionBarFragment.java]: do you need to create the args bundle, even if the fragment doesn't need any args?

File: app/src/main/java/com/rage/clamber/Fragments/UserInfoDialogFragment.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    /**
+     * Method to inflate and create a dialog window. Method is called from within the
+     * UserInfoActivity to gather general user information.
+     * @return returns the Dialog fragment.
+     */
+    @Override
+    public Dialog onCreateDialog(Bundle savedInstanceState) {
+
+        View rootView = LayoutInflater.from(getActivity()).inflate(R.layout.fragment_user_info_dialog, null);
```
Comment (nonvirtualthunk) [inline app/src/main/java/com/rage/clamber/Fragments/UserInfoDialogFragment.java:47]: I don't think this is wrong, just wondering, what does the `null` as a second argument to `inflate()` indicate?

File: app/src/main/java/com/rage/clamber/Activities/HomePage.java (hunk 1 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Activities/HomePage.java:14]: Good idea - I will do that!

File: app/src/main/java/com/rage/clamber/Fragments/ActionBarFragment.java (hunk 2 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Fragments/ActionBarFragment.java]: It doesn't look like I would ever need to pass in arguments so I will remove this.

File: app/src/main/java/com/rage/clamber/Fragments/UserInfoDialogFragment.java (hunk 3 above)
Comment (rachelbock) [inline app/src/main/java/com/rage/clamber/Fragments/UserInfoDialogFragment.java:47]: It is the parent ViewGroup which is an optional parameter. It is not necessary in this instance as there is no parent.

**Comment by nonvirtualthunk** [comment]
looks good, merging

**Pull request merged**